src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -103,6 +103,10 @@ def load_arguments(self, _):
     bicep_outdir_type = CLIArgumentType(options_list=['--outdir'], completer=DirectoriesCompleter(), help="When set, saves the output at the specified directory.")
     bicep_outfile_type = CLIArgumentType(options_list=['--outfile'], completer=FilesCompleter(), help="When set, saves the output as the specified file path.")
     bicep_stdout_type = CLIArgumentType(options_list=['--stdout'], action='store_true', help="When set, prints all output to stdout instead of corresponding files.")
+    bicep_indentkind_type = CLIArgumentType(options_list=['--indent-kind'], action='store_true', help="Set indentation kind. Valid values are ( Space | Tab ).")
+    bicep_indentsize_type = CLIArgumentType(options_list=['--indent-size'], action='store_true', help="Number of spaces to indent with (Only valid with --indent-kind set to Space).")
```

Comment:
> -  FAIL - HIGH severity: faulty_help_example_parameters_rule
    Help-Entry: `bicep format` - 
	There are 2 violations:
	"az bicep format --file {bicep_file} --indent-kind {indent_kind}" is not a valid command.
	unrecognized arguments: {indent_kind}
	"az bicep format --file {bicep_file} --indent-size {indent_size}" is not a valid command.
	unrecognized arguments: {indent_size}

I believe the CI issue is caused by your wrong usage of `action='store_true'`. Please note that the `action='store_true'` is only used for parameters representing switches, and it does not accept any values. For more details, please refer to the doc [registering-flags](https://github.com/Azure/azure-cli/blob/29767d75d850ddc1c24cc85bd46d861b61d77a47/doc/authoring_command_modules/authoring_commands.md#registering-flags)
```suggestion
    bicep_indentkind_type = CLIArgumentType(options_list=['--indent-kind'], help="Set indentation kind. Valid values are ( Space | Tab ).")
    bicep_indentsize_type = CLIArgumentType(options_list=['--indent-size'], help="Number of spaces to indent with (Only valid with --indent-kind set to Space).")
```